VAMobile/src/api/demographics/queryKeys.ts

```diff
@@ -0,0 +1,4 @@
+export const demographicsKeys = {
```

Comment:
The query keys for demographics are slightly different from what a [typical query key factory](https://tkdodo.eu/blog/effective-react-query-keys#use-query-key-factories) would look like. Typically, you would have an `all` property that holds the base key that other keys in the factory add to their array. This way, you could easily invalidate all queries related to a feature. For gender identity, I'm not sure that we have a case where we would need to invalidate both `demographics` and `genderIdentityOptions`, so I omitted `all` for this feature. If that changes, we can always add it here